source/xlsxreader.d

```diff
@@ -1039,7 +1245,7 @@ string specialCharacterReplacementReverse(string s) {
 		.replace("&amp;", "&");
 }
 
-unittest {
+@trusted unittest {
```

Comment:
`@trusted ` somewhat changes nothing. It doesn't test if it is `@safe` nor does it state that it is unsafe. Please remove the `@trusted` from the unittests

Reply:
They are needed because of the `@safe:` at the top. So shall I convert that @safe: to individual (member) function qualifiers?

Reply:
maybe then `@safe`at the top is not good.